### ISSUE TITLE
Ruffle Error on http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend

### ISSUE THREAD

**Issue by nalply** [ruffle-rs/ruffle#2049]
This is a weather site and it uses Flash to display diagrams, for example temperature trends the next two weeks. It didn't work with Ruffle, however. I installed Ruffle as a temporary extension as proposed here: https://ruffle.rs/#usage

# Error Info
Error name: TypeError
Error message: bi[e] is not a function
Error stack:
```
hi@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:32170
Function.prototype.bind/<@http://www.meteocentrale.ch/fileadmin/template/prototype.js:105:21
promise callback*i@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:5400
s/c</<@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:5693
s/c<@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:5429
s@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:5716
x@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:14079
C@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:24816
fromNativeObjectElement@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:26927
U@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:29431
polyfill/</<@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:31225
MutationCallback*polyfill/<@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:31154
polyfill@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:31281
init@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:4437
EventListener.handleEvent*o@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:3830
negotiate@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:5098
892@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:31010
hi@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:32170
@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:64442
@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:64451
@http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend:3:64455

```

# Player Info
Allows script access: true
Player type: Object
SWF URL: http://data.meteomedia.de:80/details/anychart/swf/Chart.swf
Param movie: http://data.meteomedia.de:80/details/anychart/swf/Chart.swf
Param bgcolor: #FFFFFF
Param allowScriptAccess: always
Param flashvars: __externalobjid=__AnyChart___0&xmlfile=http://data.meteomedia.de:80/details/AnychartData.php?wmo1=067350&type=Temps&lang=de&offset=-1&__enableevents=1&preloaderInitText=Initializing... &preloaderLoadingText=Loading... &initText=Initializing...&xmlLoadingText=Loading xml...&templatesLoadingText=Loading templates...&resourcesLoadingText=Loading resources...&nodatatext=No data&waitingfordatatext=Waiting for data...
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined
Attribute 3: undefined
Attribute 4: undefined
Attribute 5: undefined
Attribute getNamedItem: undefined
Attribute setNamedItem: undefined
Attribute removeNamedItem: undefined
Attribute item: undefined
Attribute getNamedItemNS: undefined
Attribute setNamedItemNS: undefined
Attribute removeNamedItemNS: undefined
Attribute length: undefined

# Page Info
Page URL: http://www.meteocentrale.ch/de/europa/schweiz/wetter-adelboden/details/N-3512607/#trend

# Browser Info
Useragent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:85.0) Gecko/20100101 Firefox/85.0
OS: MacIntel

# Ruffle Info
Version: 0.1.0
Name: nightly 2020-12-20
Channel: nightly
Built: 2020-12-20T00:35:32.589Z
Commit: 8a0e55d74a092dc4f85ceb74ff5e46b289e47f50

**Comment by Toad06** [comment]
Fixed by #2051 (the file itself will not load due to CORS issues).

**Comment by Poopooracoocoo** [comment]
I'm still seeing CORS errors in the console :/

**Comment by nalply** [comment]
> Toad06 wrote:
> Fixed by #2051 (the file itself will not load due to CORS issues).

Thanks!

> Poopooracoocoo wrote:
> I'm seeing CORS errors in the console with 2020-12-23 :/

It seems that CORS issues are difficult to fix, perhaps because Ruffle is ran by an extension and Flash by a native plugin, just guessing here... See https://github.com/ruffle-rs/ruffle/issues/1824
